sdk/servicebus/azure-servicebus/tests/async_tests/test_queues_async.py

```diff
@@ -991,7 +989,7 @@ async def test_async_queue_schedule_multiple_messages(self, servicebus_namespace
                     message_id_b = uuid.uuid4()
                     message_b = Message(content)
                     message_b.properties.message_id = message_id_b
-                    tokens = await sender.schedule(enqueue_time, message_a, message_b)
+                    tokens = await sender.schedule([message_a, message_b], enqueue_time)
```

Comment:
You're gonna throw something heavy at me for asking this now, but this test made me think to myself, "that's actually a very sane call pattern." (To put the enqueue_time first, and allow a *kwargs of the mesage list)  I imagine you thought through the tradeoffs already, am curious to your reasoning one way or the other before I start making off-the-cuff suggestions.

Reply:
HaHa, that's true, I fight against myself and then come to this, my point would be:

1) consistent with current api (and we don't support this *args operation in SB or EH)
2) avoid extra step for scheduling batch: think about doing batch scheduling in real applications, suppose you have created 20 messages to be scheduled, if the `schedule(*args)` patter is applied:
```python
message1 = Message()
...
message20 = Message()
sender.schedule(utc_time, message1, message2, ..., message20) # You definitely won't want it this way
# Instead, you would probably still construct an array first, putting them all into an array, then schedule them
arry = [ Message() for _ in range(20) ]
sender.schedule(schedule, *array)  # you put into array, and then expand them through aterisk operation
```
So why not just accept an array instead of *args?
3. `*args` does provide flexibility when scheduling small amount of messages like just two or three, but array could also achieve this with ease: `sender.schedule(utc_time, [message1, message2])`

Reply:
++ Reasonable arguments.  Especially if we've set precedent with our other SDKs I don't think I'd change this, I will make a note about validating this approach in our user studies however.